packages/ember-routing-htmlbars/tests/helpers/closure_action_test.js

```diff
@@ -312,20 +312,20 @@ test('action value is returned', function(assert) {
 test('action should be called on the correct scope', function(assert) {
   assert.expect(1);
 
-  innerComponent = EmberComponent.extend({
+  innerComponent = run(EmberComponent.extend({
     fireAction() {
       this.attrs.submit();
     }
-  }).create();
+  }), 'create');
 
-  outerComponent = EmberComponent.extend({
+  outerComponent = run(EmberComponent.extend({
     layout: compile('{{view innerComponent submit=(action outerSubmit)}}'),
     innerComponent,
     isOuterComponent: true,
     outerSubmit() {
       assert.ok(this.isOuterComponent, 'action has the correct context');
     }
-  }).create();
+  }), 'create');
```

Comment:
Although I'm undoing this, `extend` is not using the run loop here. The result of `extend` is being passed as the context of the run loop on which `'create'` is looked up/invoked.